pkg/status/status.go

```diff
@@ -372,24 +264,17 @@ func NewReporter(cfg *rest.Config, mgr manager.Manager, namespace string, name s
 	if err != nil {
 		return nil, fmt.Errorf("failed to create raw client: %s", err.Error())
 	}
-	syncRatio, err := NewSyncRatio(successRatio, syncsBeforeTruncate, syncTruncateValue)
-	if err != nil {
-		return nil, fmt.Errorf("failed to create status sync ratio: %s", err.Error())
-	}
 
 	// If version is an empty string, warn that the operator is not a part of the OpenShift release payload.
 	if version == "" {
 		version = "OpenShift Independent Version"
 	}
 
 	return &reporter{
-		configClient: configClient,
-		rawClient:    rawClient,
-		namespace:    namespace,
-		version:      version,
-		syncRatio:    syncRatio,
-		// Add a buffer to prevent dropping syncs
-		syncCh:              make(chan error, 25),
```

Comment:
Do we need the receiver on [line 405](https://github.com/operator-framework/operator-marketplace/pull/323/files#diff-a5666326aefdf43a8e89fd4871449fa0L405)

Reply:
Not sure what that is linking to: https://github.com/operator-framework/operator-marketplace/blob/8ffe135acd11523609bc2dd3ef1c5312a6111f11/pkg/status/status.go
The receiver has been ripped off since we're not keeping track of any syncs anymore.

Reply:
https://github.com/operator-framework/operator-marketplace/blob/8ffe135acd11523609bc2dd3ef1c5312a6111f11/pkg/status/status.go#L290

Reply:
Absolutely, good catch!